object/map.go

```diff
@@ -177,7 +177,8 @@ func (m *Map) GetAttr(name string) (Object, bool) {
 			},
 		}, true
 	}
-	return nil, false
+	o, ok := m.items[name]
+	return o, ok
```

Comment:
😄 

I almost did this at some point but hesitated at the time. I still have mixed feelings on it. It's definitely _usually_ a nice usability improvement but with the downside that it's going to confuse the developer when/if the map contains a key that's the same as one of the built-in method names.

Probably most common would be:

```json
{ "keys": [1, 2, 3] }
```

and

```json
{ "items": [1, 2, 3] }
```

You have me leaning towards rolling with this update though. Thoughts?

Reply:
I suppose Javascript has a similar situation. Not that that makes it a good thing necessarily.

```
$ node
Welcome to Node.js v21.4.0.
Type ".help" for more information.
> x = {foo: 42}
{ foo: 42 }
> x.foo
42
> x.hasOwnProperty
[Function: hasOwnProperty]
```

Reply:
yeah, I though about that, I'm ok to revert this if you think it's better to leave it was it was before.

Reply:
Let's go ahead with this change. I'm favoring the added convenience.

We should do the same on the setter. Up to you whether we do that in this PR.

Reply:
I can add it.

Reply:
added.